UnitTestsGordon360/AuthenticationTests.cs

```diff
@@ -0,0 +1,49 @@
+using Gordon360;
+using Gordon360.Static.Names;
+using Gordon360.Authorization;
+
+// Tests function names are made of three parts:
+//   1. The name of the method being tested
+//   2. The scenario under which it's being tested
+//   3. The expected behavior when the scenario is invoked
+//
+// Body of test should have three parts:
+//   1. Arrange your objects, create and set them up
+//   2. Act on an object
+//   3. Assert that something is as expected
+
+namespace UnitTestsGordon360
+{
+    public class AuthenticationTests
+    { 
+        [Theory]
+        [InlineData("A student news resource")] // Resource.NEWS
+        [InlineData("Slider to be shown on the homepage")] // Resource.SLIDER
```

Comment:
Is it possible to use the static names directly here? That would be less error-prone than having to copy the string, plus it ensures that they stay synchronized.

Reply:
That makes sense! Just fixed it :O